packages/utils/src/customer-requests/medical-record-brief-input.ts

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable */
+// @ts-nocheck
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { Bundle, Resource } from "@medplum/fhirtypes";
+// import { summarizeFilteredBundleWithAI } from "./medical-record-brief-input";
+import { summarizeFilteredBundleWithAI } from "@metriport/core/external/aws/lambda-logic/bundle-to-brief";
+import { bundleToHtml } from "@metriport/core/external/aws/lambda-logic/bundle-to-html";
+import fs from "fs";
+import { uuidv7 } from "../shared/uuid-v7";
+
+/**
+ * Script to trigger MR Summary generation on a FHIR payload locally, with the AI Brief included in it.
+ */
+
+// Update this to staging or local URL if you want to test the brief link
+const dashUrl = "http://dash.metriport.com";
+
+const SOURCE_DIR = "";
+const SOURCE_PATIENT_ID = "";
+const SOURCE_BUNDLE_FILE = `${SOURCE_PATIENT_ID}.json`;
+
+async function main() {
+  // TODO: Condense this functionality under a single function and put it on `@metriport/core`, so this can be used both here, and on the Lambda.
```

Comment:
nit: tie to an issue

Reply:
I'll add this to the last PR to minimize rebasing conflicts

Reply:
done